revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                        <a href="/signin/?next=/project/1/&amp;reason=donate#login" id="donate-button" class="btn-blue btn-i-want-to-donate">
+                                                            <p>DONATE</p>
+                                                        </a>
+                                                    </div>
+                                                </div>
+                                                </div>
+                                            </div>
+                                            <div class="info-main">
+                                                <div class="blue-bar">
+                                                    <span class="pull-left">
+                                                        <span class="bold">${{ project.amount_donated|floatformat:"0"|intcomma }}</span> Donated
+                                                    </span>
+                                                    <span class="pull-right">
+                                                        <span class="bold">{{ project.days_left }}</span> Days Left
+                                                    </span>
+                                                </div>
+                                                <!-- end .blue-bar -->
+                                                <div class="dark-blue-bar">
+                                                    <span class="pull-left actual-energy">
+                                                        <span class="bold">{{ project.actual_energy }}</span> lbs CO
+                                                        <sub>2</sub> Avoided
+                                                    </span>
+                                                    <span class="pull-right">
+                                                        <span class="bold">{{ donor_count }}</span> {{ settings.revolv_cms.ProjectPageSettings.donors_wording }}
+                                                    </span>
+                                                </div>
+                                                <!-- end .blue-bar -->
+                                            </div>
+                                            <!-- end .info-main -->
+                                            <!-- end .video-main -->
+                                        </div>
+                                    </div>
+                                    <!-- end .info-section -->
+                                </div>
+                                <!-- end .details-active-project-module -->
+                                <!-- basic $ 20 div start here -->
```

Comment:
This comment is not needed, please delete it or update it it to describe the HTML that follows.